### ISSUE TITLE
Run also app-level tests

### ISSUE THREAD

**Issue by mitar** [tozd/meteor-test-runner#8]
Meteor has now a concept of app-level tests. We should run them as well.

**Comment by zeroasterisk** [comment]
Yes - I got really excited when I saw this script, I like the approach a lot!

But now that we don't split apps up into packages, and rely on imports/exports... It's a bit more useful to rely on the "official" testing approach recommended in the guide...

But it's still a pain to setup a testing script on circleci (or not) and I think what you've got going here is a good option.

**Comment by mitar** [comment]
I still have apps structured in packages. I think it gives some easier way to turn on/off features. (You just control at the top-level which packages you want to include.)

Personally, I have not yet had time to look into app-level testing. If you are interested in contributing a pull request to this repository with app-level tests as well, that would be great.

I read the guide, but I do not really see how exactly I would end up doing them with same ease as tinytest were.
